lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -39,6 +40,10 @@ def fragment_cache(cached_hash, non_cached_hash)
           JsonApi::FragmentCache.new().fragment_cache(root, cached_hash, non_cached_hash)
         end
 
+        def default_key_format
+          :dasherize
```

Comment:
yep that would be cool :smile: not required for the PR itself, but I'm curious about it. That remembers me the Benchmark PR I had, I might need to pick that up again.